tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
On second thought, I'm not sure we should do any sort of sanitization or escaping in our custom error handler. It's possible that our method of sanitization could break client code. I believe it's the client's responsibility ensure the safety of the strings they are rendering to HTML.